### PR TITLE
38 test registration endpoint

### PR DESCRIPTION
**Pull request by AnthonyHewins**
first of our test implemented.
only thing you need to check is the server.py file and models.py, the rest is just additions

### CODE REVIEW

File: server/models.py [hunk 1]
```diff
@@ -5,36 +5,50 @@
...
-        except BadSignature:
-            return None  # invalid token
-        user = User.query.get(data['id'])
-        return user
+from datetime import datetime
+
+# All models inherit from this. It contains the fields we
+# need for important data
+class Model(Document):
+  updated = DateTimeField(default=datetime.now())
```
Comment (kdjohnson) [inline server/models.py]: Indentation should be 4 spaces not 2.

File: server/server.py [hunk 2]
```diff
@@ -47,113 +56,115 @@
...
-    hashed = bcrypt.hashpw(password.encode(), bcrypt.gensalt())
-    user_datastore.create_user(email=email, password=hashed)
-
-    # TODO: error handle this and if it doesn't work do something else besides the success in jsonify
-    #send_email(recipients=[email], subject="ay whaddup", text="Hello from AR-top")
-
-    return jsonify(success="Account has been created! Check your email to validate your account.")
-
+  # Confirm the request
+  email, password = None, None
```
Comment (kdjohnson) [inline server/server.py]: Indentation should be 4 spaces not 2.

File: server/models.py (hunk 1 above)
Comment (AnthonyHewins) [reply server/models.py]: ah i thought we said 2, ill change it back

**Comment by joshherkness** [comment]
Is this done, can it be pulled in and closed out?

**Comment by AnthonyHewins** [comment]
Yeah its done just need approvals

**Pull request merged**